index.html

```diff
@@ -1,14 +1,82 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Page Title</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" media="screen" href="styles.css">
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>HTML Forms</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
     <script src="main.js"></script>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <!-- NAV -->
+    <nav>
+      <div class="navbar-container">
+        <div class="navbar-text">
+          <div>Already have an Intuit account?</div>
+          <a href="#">I forgot my User ID or Password</a>
+        </div>
+        <div class="navbar-button"><button>Sign In</button></div>
+      </div>
+    </nav>
+
+    <!-- HEADER -->
+    <header>
+      <h1>INTUIT</h1>
```

Comment:
Recommendation: You can improve looks of the header by adding the logo of Intuit instead of using text.